scRNA-seq-advanced/setup/rms/integrate_rms.R

```diff
@@ -0,0 +1,131 @@
+# load training modules R proj
+training_modules_dir <- file.path(here::here(), "..")
+renv::load(training_modules_dir)
```

Comment:
Okay, I added this because I was getting errors that `batchelor` wasn't installed even though it's in the lock file, but I can remove this with the assumption that we would have it installed.

Reply:
yeah, you may have to install it yourself for now, but we will make sure everybody has it.